opsdroid/tests/test_logging.py

```diff
@@ -174,26 +164,40 @@ def test_configure_default_logging(capsys):
     opsdroid.configure_logging(config)
     rootlogger = logging.getLogger()
     assert len(rootlogger.handlers), 2
-    assert isinstance(rootlogger.handlers[0], logging.StreamHandler)
+    assert isinstance(rootlogger.handlers[0], logging.handlers.RotatingFileHandler)
     assert rootlogger.handlers[0].level == logging.INFO
-    assert isinstance(rootlogger.handlers[1], logging.handlers.RotatingFileHandler)
+    assert isinstance(rootlogger.handlers[1], RichHandler)
```

Comment:
wait. what? if `console: true` then we end up with a `RichHandler`?

Reply:
You can have both on, the Rich logging will take precedence by default, but you might want to have console logging to log to a file for example